Classes/Views/IssueTextActionsView+Markdown.swift

```diff
@@ -31,7 +31,24 @@ extension IssueTextActionsView {
             IssueTextActionOperation(icon: UIImage(named: "bar-ul"), operation: .line("- ")),
             IssueTextActionOperation(icon: UIImage(named: "bar-indent"), operation: .line("  ")),
             IssueTextActionOperation(icon: UIImage(named: "bar-link"), operation: .wrap("[", "](\(UITextView.cursorToken))")),
-            ]
+            IssueTextActionOperation(icon: UIImage(named: "cloud-upload"), operation: .execute({ [weak viewController] in
```

Comment:
Doesn't this show up in `IssueViewController`? But it doesn't conform to the delegate or implement any of the delegate stuff to handle succesful upload. So wont the button show up but do nothing?